### ISSUE TITLE
Test the translation utility on sims converted to ES6

### ISSUE THREAD

**Issue by jbphet** [phetsims/chipper#883]
String handling is going to be changed fairly significantly when we do the conversion of all sim code to ES6.  The translation utility (aka Rosetta) relies on extraction of the strings used by the sim from the built HTML5 file.  We should verify that this process works when there are converted and built ES6 sims available.

The parent issue for this is https://github.com/phetsims/chipper/issues/820.

An issue that is related, since it has to do with string changes necessary for ES6 conversion, is https://github.com/phetsims/chipper/issues/832.

**Comment by jbphet** [comment]
Assigning to @jonathanolson, who is going to produce the first fully converted and built ES6 sims.  Once there are several of these available, this issue should be assigned back to me with links to these built sims, and I'll do the testing.

**Comment by samreid** [comment]
Marking high priority since this should be completed before migration.

**Comment by jonathanolson** [comment]
Built sims should be available under https://phet-dev.colorado.edu/modules-testing/

**Comment by jonathanolson** [comment]
https://phet-dev.colorado.edu/modules-testing2/ includes the patched version of molarity (with the new string keys) and should be used instead.

**Comment by jbphet** [comment]
I tested this, and the tests failed.  After doing some digging, the reason appears to be that while the variable `window.phet.chipper.strings` is still available, the data structure with the list of everything used in the string plugin is not, and Rosetta was relying on this.  I think it can be converted reasonably easily.

**Comment by jbphet** [comment]
Modifications complete, changes deployed, see https://github.com/phetsims/rosetta/issues/224 for details.  I'll try to remember to test this on the first post-ES6 migration publication (I'll open a separate issue for it when I hear of something that is getting close).  I'll close this one so we know that it isn't blocking the ES6 migration.
